example/satosa/pyeudiw_backend.yaml

```diff
@@ -258,3 +258,19 @@ config:
             db_trust_sources_collection: trust_sources
             data_ttl: 63072000 # 2 years
           # - connection_params:
+
+  credential_presentation_handlers:
+    - format: "dc+sd-jwt"
+      module: "pyeudiw.sd_jwt.verifier"
+      class: "SDJWTVerifier"
+    - format: "ldp_vp"
+      module: "pyeudiw.openid4vp.presentation_submission"
+      class: "LdpVp"
+    - format: "jwt_vp_json"
+      module: "pyeudiw.openid4vp.presentation_submission"
+      class: "JwtVpJson"
```

Comment:
```suggestion
```

please check the presence of each classes within the init of the backend, to avoid unexpected exceptions runtime